### DISCUSSION TITLE
Hide Menu items

### DISCUSSION THREAD

**Discussion by BLUESALOON** [egzumer/uv-k5-firmware-custom#411] [Ideas]
Would it be possible to only show selected menu items to the user?
For example...
If I give a radio to a member of staff, only the menu items they actually need will show in the menu. 
The numbers assigned to the menu items would remain unchanged.  A scroll through the menu could look something like...
  2.60   TxPwr
36.60   BackLt
54.60   D List
60.60   Sql

Thanks for all the hard work.

**Comment by egzumer** [comment]
It is possible if there would be someone willing to implement this functionality :)
This is only useful in commercial environment.
